install/migrations/update_10.0.2_to_10.0.3/ticket.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2022 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+/**
+ * @var DB $DB
+ * @var Migration $migration
+ */
+
+if (!$DB->fieldExists("glpi_tickets", "takeintoaccountdate")) {
+    $migration->addField("glpi_tickets","takeintoaccountdate","timestamp",['null' => true,'after' => 'solvedate']);
+    $migration->addKey("glpi_tickets", "takeintoaccountdate");
+    $migration->migrationOneTable("glpi_tickets");
+
+    $tickets_iterator = $DB->request([
+        'SELECT'    => ['id'],
+        'FROM'      => 'glpi_tickets',
+        'WHERE'     => [
+            'takeintoaccount_delay_stat' => ['>', 0],
+        ]
+    ]);
+
+    foreach ($tickets_iterator as $data) {
+       $ticket = new Ticket();
+       $ticket->getFromDB($data['id']);
+       $tia_delay = $ticket->getField('takeintoaccount_delay_stat');
+       $ticket_date = $ticket->getField('date');
```

Comment:
`getField` is for display purpose only, as it may return `N/A`.
```suggestion
       $tia_delay = $ticket->fields['takeintoaccount_delay_stat'];
       $ticket_date = $ticket->fields['date'];
```